### PR TITLE
Completed imputs with different afferents and intersection part

### PR DESCRIPTION
**Pull request by karan-modh**
Input Format:
V_*
T_*
A_*
O_*
G_*

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def main():
 		
 		for i in range(len(new_inputs)):
 			# get the sense of input (V/T/O/G/A)
-			sense = new_inputs[i][0]
+			sense, new_input, intensity = new_inputs[i].split(':')
```
Comment (Itisha2987) [inline main.py]: What is the new input format?
Are we taking input as T:hot:4?[1] or T_hot:4?[2] If [2] is followed than "sense, new_input, intensity = new_inputs[i].split(':')" this won't work.

File: main.py [hunk 2]
```diff
@@ -99,7 +99,8 @@ def main():
 			observations_for_given_level.update(raised_observations)
 
 		# Adding old observations to input set
-		inputs = list(observations_for_given_level)
+		# inputs = list(observations_for_given_level)
+		inputs = [observation+":1" for observation in list(observations_for_given_level)]
```
Comment (Itisha2987) [inline main.py:105]: You are taking intensity as 1, right??

Comment (karan-modh) [reply main.py:105]: yes

File: main.py (hunk 1 above)
Comment (karan-modh) [reply main.py]: I think, there's one more commit after this change.
I've mentioned new input format in code as well.
T:hot:4 is the new input format

**Pull request merged**